### PR TITLE
send account confirmation emails to relevant users

### PR DESCRIPTION
**Pull request by czue**
https://docs.google.com/document/d/1gZlOGroRTeUMHXwzCuOVw2EUuJcjOsAaAe1qc2iUEe0/edit#

##### SUMMARY

Hooks up the actual email invitation part of the user confirmation workflow. This should make the whole thing "feature complete" for a V1. Planning on running through the flow and adding some docs tomorrow (as well as responding to other PR feedback and remaining todos).

Review by commit - first one is just some cleanup.

### CODE REVIEW

File: corehq/apps/registration/templates/registration/email/mobile_worker_confirm_account.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% extends "registration/email/confirm_account.html" %}
+{% load i18n %}
+
+{% block preview_text %}
+  {% blocktrans %}
+    You have been added to the {{ domain }} project on CommCare HQ.
```
Comment (orangejenny) [inline corehq/apps/registration/templates/registration/email/mobile_worker_confirm_account.html]: Extract "CommCare HQ", like in [web_users.html](https://github.com/dimagi/commcare-hq/blob/985100b7b6b479b7180d5d2543d351c948680330/corehq/apps/users/templates/users/web_users.html#L14)

**Review by orangejenny — APPROVED** [review]

**Comment by orangejenny** [comment]
Is QA going to sanity test this?

**Comment by orangejenny** [comment]
(this whole feature, not this specific PR)

**Comment by czue** [comment]
> Is QA going to sanity test this?

Good question. I hadn't asked anyone (I think it is planned to be used in our US COVID project very soon) so it may just end up being easier to informally test that way? Though I could request a test if you think that's a good idea.

**Comment by czue** [comment]
(will address feedback tomorrow)

**Comment by orangejenny** [comment]
It's worth mentioning to Ben; he was able to rearrange other QA requests last week for the sake of testing other covid-related work.

File: corehq/apps/registration/templates/registration/email/mobile_worker_confirm_account.html (hunk 1 above)
Comment (czue) [reply corehq/apps/registration/templates/registration/email/mobile_worker_confirm_account.html]: https://github.com/dimagi/commcare-hq/pull/26946/commits/6d83722c0ae553d13a322ff906d045acd8a99720

**Pull request merged**